src/hearings/containers/hearing-actuals/hearing-actuals-timing/hearing-actuals-timing.component.ts

```diff
@@ -0,0 +1,119 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { Store } from '@ngrx/store';
+import { Subscription } from 'rxjs';
+import { filter } from 'rxjs/operators';
+import { HearingActualsMainModel } from '../../../models/hearingActualsMainModel';
+import { HearingActualsStateData } from '../../../models/hearingActualsStateData.model';
+import { HearingsService } from '../../../services/hearings.service';
+import * as fromHearingStore from '../../../store';
+import { ValidatorsUtils } from '../../../utils/validators.utils';
+
+@Component({
+  selector: 'exui-hearing-actuals-timing',
+  templateUrl: './hearing-actuals-timing.component.html',
+  styleUrls: ['./hearing-actuals-timing.component.scss']
+})
+export class HearingActualsTimingComponent implements OnInit, OnDestroy {
+
+  public static timeMatcher = /\d{2}:\d{2}/;
+  public form: FormGroup;
+  private hearingActuals: HearingActualsMainModel;
+  private sub: Subscription;
+
+  public constructor(private readonly fb: FormBuilder,
+                     private readonly hearingStore: Store<fromHearingStore.State>,
+                     private readonly router: Router,
+                     private readonly hearingsService: HearingsService,
+                     private readonly validatorsUtils: ValidatorsUtils,
+  ) {
+  }
+
+  private static getStartTime(hearingActuals: HearingActualsMainModel): string {
+    const plannedTime = hearingActuals.hearingPlanned.plannedHearingDays[0].plannedStartTime;
+    const actualTime = hearingActuals.hearingActuals.actualHearingDays[0].hearingStartTime;
+    return actualTime ? actualTime.match(this.timeMatcher)[0] : plannedTime.match(this.timeMatcher)[0];
```

Comment:
would be better to use moment js to format the time, i.e. if the planned daytime is from the iso date with timezone, this time will be incorrect with Regex.

Reply:
Agreed.